packages/main/cypress/specs/MultiInput.cy.tsx

```diff
@@ -218,45 +212,35 @@ describe("MultiInput Web Component", () => {
 			.should("have.been.calledTwice");
 	});
 })
+
 describe("MultiInput tokens", () => {
-	it("adds a token to multi input", () => {
+
+	it("Simulate adding token to the multiinput", () => {
 		cy.mount(
 			<>
 				<MultiInput id="single-token">
-					<Token slot="tokens" text="Amet"></Token>
+					<Token slot="tokens" text="Amet" />
 				</MultiInput>
 				<Button id="add-to-single">Add more tokens</Button>
 			</>
 		);
 
-		cy.get("ui5-multi-input")
-			.as("multiInput");
+		const addTokenSpy = cy.stub().as("addTokenSpy");
 
-		cy.get("ui5-button")
-			.then(button => {
-				button[0].addEventListener("click", () => {
-					addTokenToMI(createTokenFromText("test"), "single-token");
-				});
+		cy.get("[ui5-button]").then(button => {
+			button[0].addEventListener("click", () => {
+				addTokenSpy();
```

Comment:
With this change you call the spy when the button is clicked. It tests that the button was clicked once. 
I think you can either delete this test or keep the old test that uses the createTokenFromText and addTokenToMI methods. 
We do not fire specific event to inform the application that a new token should be created. The apps listen for "change" event and decide to create or not a tokens from the input.